### PR TITLE
[165] add billing contact email and phone

### PR DESCRIPTION
**Pull request by caravinci**
Adds two fields to a form, and handles their data.

### CODE REVIEW

File: enrolment-service/enrol/core/migrations/0009_add_billing_contact_email_and_phone.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```
Comment (caravinci) [inline enrolment-service/enrol/core/migrations/0009_add_billing_contact_email_and_phone.py:1]: Autogenerated db migration to add the new fields, and fill the existing rows with empty strings.

File: enrolment-service/enrol/core/models.py [hunk 2]
```diff
@@ -69,8 +69,11 @@ class Meta:
 
 
 class BillingContact(models.Model):
+
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     contact_name = models.CharField(max_length=100)
+    email = models.EmailField()
+    phone = models.CharField(max_length=50)
```
Comment (caravinci) [inline enrolment-service/enrol/core/models.py:76]: new fields in the Billing Contact model

File: enrolment-service/enrol/core/serializers.py [hunk 3]
```diff
@@ -54,7 +54,7 @@ class Meta:
 class BillingContactSerializer(serializers.ModelSerializer):
     class Meta:
         model = BillingContact
-        fields = ('id', 'contact_name', 'street_address', 'city', 'region',
+        fields = ('id', 'contact_name', 'email', 'phone', 'street_address', 'city', 'region',
```
Comment (caravinci) [inline enrolment-service/enrol/core/serializers.py:57]: let the serialiser know about said new fields

File: enrolment-service/enrol/core/serializers.py [hunk 4]
```diff
@@ -75,7 +75,7 @@ def validate(self, data):
         if data['daco_email'].lower() == data['institution_email'].lower():
             raise serializers.ValidationError(
                 "Institution email must be different from daco email")
-        if data['invoice_consent'] != True:
+        if data['invoice_consent'] is not True:
```
Comment (caravinci) [inline enrolment-service/enrol/core/serializers.py:78]: conditional syntax, the python way

File: enrolment-service/enrol/core/views.py [hunk 5]
```diff
@@ -344,6 +344,8 @@ def perform_create(self, serializer):
             data = {
                 **data,
                 'Billing Contact': application.billing_contact.contact_name,
+                'Billing Email': application.billing_contact.email,
+                'Billing Phone Number': application.billing_contact.phone,
```
Comment (caravinci) [inline enrolment-service/enrol/core/views.py:348]: fields being received and passed

File: enrolment-ui/src/modules/Projects/components/ReqFormStep1.js [hunk 6]
```diff
@@ -6,90 +6,19 @@ import { RFInput, RFSelectFlat, rules } from '../../ReduxForm';
 
 import { toggleBillingFields, changeCountry, changeBillingCountry } from '../redux';
 
-class ReqFormStep1 extends Component {
```
Comment (caravinci) [inline enrolment-ui/src/modules/Projects/components/ReqFormStep1.js:9]: class component without using methods... turning into functional.

File: enrolment-ui/src/modules/Projects/components/ReqFormStep1.js [hunk 7]
```diff
@@ -6,90 +6,19 @@ import { RFInput, RFSelectFlat, rules } from '../../ReduxForm';
...
-  renderBillingFields(disabled, countryRegion, changeBillingCountry, selectedBillingRegion) {
```
Comment (caravinci) [inline enrolment-ui/src/modules/Projects/components/ReqFormStep1.js:10]: relocating function into render

File: enrolment-ui/src/modules/Projects/components/ReqFormStep1.js [hunk 8]
```diff
@@ -236,14 +165,110 @@ class ReqFormStep1 extends Component {
...
 
-        {showBillingFields
-          ? this.renderBillingFields(
-              disabled,
-              countryRegion,
-              changeBillingCountry,
-              selectedBillingRegion,
-            )
-          : null}
+        {showBillingFields && (
```
Comment (caravinci) [inline enrolment-ui/src/modules/Projects/components/ReqFormStep1.js:168]: billing contact stuff

**Review by caravinci — COMMENTED** [review]
Comments to help review

**Review by ciaranschutte — APPROVED** [review]

**Pull request merged**